client/jetpack-onboarding/constants.js

```diff
@@ -28,17 +28,23 @@ export const JETPACK_ONBOARDING_STEPS = {
 };
 
 export const JETPACK_ONBOARDING_SUMMARY_STEPS = {
-	SITE_TITLE_DESCRIPTION: translate( 'Site Title & Description' ),
-	SITE_TYPE: translate( 'Type of Site' ),
-	HOMEPAGE_TYPE: translate( 'Type of Homepage' ),
-	CONTACT_FORM: translate( 'Contact Us Form' ),
 	JETPACK_CONNECTION: translate( 'Jetpack Connection' ),
 	THEME: translate( 'Choose a Theme' ),
 	SITE_ADDRESS: translate( 'Add a Site Address' ),
 	STORE: translate( 'Add a Store' ),
 	BLOG: translate( 'Start a Blog' ),
 };
 
+export const JETPACK_ONBOARDING_STEP_TITLES = {
+	[ JETPACK_ONBOARDING_STEPS.SITE_TITLE ]: translate( 'Site Title & Description' ),
+	[ JETPACK_ONBOARDING_STEPS.SITE_TYPE ]: translate( 'Type of Site' ),
+	[ JETPACK_ONBOARDING_STEPS.HOMEPAGE ]: translate( 'Type of Homepage' ),
+	[ JETPACK_ONBOARDING_STEPS.CONTACT_FORM ]: translate( 'Contact Us Form' ),
+	[ JETPACK_ONBOARDING_STEPS.BUSINESS_ADDRESS ]: translate( 'Business Address' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add a business address.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).